android/engine/src/main/java/org/smartregister/fhircore/engine/di/FhirEngineModule.kt

```diff
@@ -68,14 +70,31 @@ class FhirEngineModule {
                 headersToIgnore = listOf(AUTHORIZATION, COOKIE)
               )
             ) { Timber.tag(QUEST_OKHTTP_CLIENT_TAG).d(it) }
-        )
+        ),
+        customSearchParameters = getCustomSearchParameters()
       )
     )
-
+    Timber.e("Fhirengine module custom params set +++++++++++++++++++")
     return FhirEngineProvider.getInstance(context)
   }
 
+  fun getCustomSearchParameters(): List<SearchParameter> {
+    val activeGroupSearchParameter =
+      SearchParameter().apply {
+        url = "http://example.com/SearchParameter/group-active"
+        addBase("Group")
+        name = GROUP_ACTIVE_SEARCH_PARAM
+        code = GROUP_ACTIVE_SEARCH_PARAM
+        type = Enumerations.SearchParamType.TOKEN
+        expression = "Group.active"
+        description = "Search the active field"
+      }
+
+    return listOf(activeGroupSearchParameter)
+  }
+
   companion object {
     private const val QUEST_OKHTTP_CLIENT_TAG = "QuestOkHttpClient"
+    const val GROUP_ACTIVE_SEARCH_PARAM = "activeGroup"
```

Comment:
For consistency I would suggest we call this `active` as the name of the property in the [FHIR Spec Group resource](https://build.fhir.org/group.html)

```suggestion
    const val ACTIVE_SEARCH_PARAM = "active"
```